src/dmd/expressionsem.d

```diff
@@ -77,7 +77,6 @@ import dmd.utf;
 import dmd.utils;
 import dmd.visitor;
 
-enum LOGSEMANTIC = false;
```

Comment:
perhaps leave this as `// debug = LOGSEMANTIC;`

Reply:
Done